README.md

```diff
@@ -44,9 +48,9 @@ This runs the command:
 docpad deploy-ghpages --env static
 ```
 
-*WARNING* `deploy` will upload the site to the repository's `origin`. If you have cloned
+_**WARNING:** Deploying will upload the site to the `deploy` branch of the `origin` remote. If you have cloned
 from the production repository and you have push access, you will actually run the docs publication
-workflow against the live production branch, whichever branch you happen to be working on.
+workflow against the live production branch, regardless of whichever branch you happen to be working on._
 
```

Comment:
I wonder if it's worth mentioning that you will need permissions to push in order to affect any changes. Otherwise nothing will happen. Is this true?

I almost seems obvious, but I wonder also if it's worth mentioning that the deploy mechanic is only applicable to the Master as it has the hook. I wonder if someone will get the impression that deploying to their own fork will somehow affect the production site.

Reply:
@jhung i do mention further up in the readme about the deployment process requiring the generated site be pushed to the `deploy` branch of the project repo. Is that sufficient?

Reply:
Is it enough is what I'm wondering or does it need to be more explicit? I honestly don't know the answer. If you all think it's fairly obvious by what is written there, then this PR is good to me.

Reply:
@waharnum what do you think?

Reply:
@jobara That sounds good to me - I don't think we'll be in this state of manual depoyment too much longer anyway, as we've got most of working automation to have the CI build it using docpad that I need to do some debugging on (in https://github.com/fluid-project/infusion-docs/pull/140)